templates/tauticord.xml

```diff
@@ -14,24 +14,32 @@
+    <Config Name="Display bandwidth" Target="TC_VC_BANDWIDTH" Default="False" Description="Whether to display bandwidth in voice channels" Type="Variable" Display="always" Required="false" Mask="false">False</Config>
+    <Config Name="Display local bandwidth" Target="TC_VC_LOCAL_BANDWIDTH" Default="False" Description="Whether to display local bandwidth in voice channels" Type="Variable" Display="always" Required="false" Mask="false">False</Config>
+    <Config Name="Display remote bandwidth" Target="TC_VC_REMOTE_BANDWIDTH" Default="False" Description="Whether to display remote bandwidth in voice channels" Type="Variable" Display="always" Required="false" Mask="false">False</Config>
+    <Config Name="Display Plex server status" Target="TC_VC_PLEX_STATUS" Default="False" Description="Whether to display Plex online status in a voice channel" Type="Variable" Display="always" Required="false" Mask="false">False</Config>
+    <Config Name="Display library stats" Target="TC_VC_LIBRARY_STATS" Default="False" Description="Whether to display library stats in voice channels" Type="Variable" Display="always" Required="false" Mask="false">False</Config>
+    <Config Name="Library stats category name" Target="TC_VC_LIBRARIES_CATEGORY_NAME" Default="Tautulli Libraries" Description="Name of the library stats voice channel category" Type="Variable" Display="always" Required="false" Mask="false">Tautulli Libraries</Config>
+    <Config Name="Stat library names" Target="TC_VC_LIBRARY_NAMES" Default="" Description="Comma-separated list of names of libraries to display stats of" Type="Variable" Display="always" Required="false" Mask="false" />
+    <Config Name="Seconds between library updates" Target="TC_VC_LIBRARY_REFRESH_SECONDS" Default="3600" Description="Seconds between library stats updates (5-minute minimum)" Type="Variable" Display="always" Required="false" Mask="false">3600</Config>
+    <Config Name="Discord admin IDs" Target="TC_DISCORD_ADMIN_IDS" Default="" Description="Comma-separated list of IDs of Discord users with bot admin privileges" Type="Variable" Display="always" Required="false" Mask="false" />
+    <Config Name="Stream details text channel" Target="TC_DISCORD_CHANNEL_NAME" Default="tauticord" Description="Name of Discord text channel where the bot will post" Type="Variable" Display="always" Required="true" Mask="false">tauticord</Config>
+    <Config Name="Use embeds" Target="TC_USE_EMBEDS" Default="True" Description="Whether to use embeds for Discord messages" Type="Variable" Display="always" Required="false" Mask="false">True</Config>
+    <Config Name="Allow analytics" Target="TC_ALLOW_ANALYTICS" Default="True" Description="Whether to allow anonymous analytics collection" Type="Variable" Display="always" Required="false" Mask="false">True</Config>
+    <Config Name="Timezone" Target="TZ" Default="UTC" Description="Timezone for the server" Type="Variable" Display="always" Required="false" Mask="false">UTC</Config>
```

Comment:
Redundant

Reply:
What is redundant?

Reply:
The TZ variable, unraid automatically sets it